core/src/main/java/org/apache/iceberg/util/DateTimeUtil.java

```diff
@@ -88,4 +88,43 @@ public static long microsFromTimestamptz(OffsetDateTime dateTime) {
   public static String formatTimestampMillis(long millis) {
     return DATE_FORMAT.format(LocalDateTime.ofInstant(Instant.ofEpochMilli(millis), ZoneOffset.UTC));
   }
+
+  public static String daysToIsoDate(int days) {
+    return dateFromDays(days).format(DateTimeFormatter.ISO_LOCAL_DATE);
+  }
+
+  public static String microsToIsoTime(long micros) {
+    return timeFromMicros(micros).format(DateTimeFormatter.ISO_LOCAL_TIME);
+  }
+
+  public static String microsToIsoDateTimeTz(long micros) {
```

Comment:
@rzhang10, here's a method name that wasn't fixed after my [earlier comment](https://github.com/apache/iceberg/pull/4871/files#discussion_r909039593).